xmppserver/src/main/java/org/jivesoftware/openfire/group/DefaultGroupPropertyMap.java

```diff
@@ -191,7 +210,7 @@ public K next() {
         public void remove() {
             delegate.remove();
             if (current instanceof String) {
-                deleteProperty((String)current);
+                deleteProperty((String)current, null); // FIXME by not providing the original value, some event handlers may not be able to handle this.
```

Comment:
Does this FIXME need a JIRA reference?

Reply:
Ack. I've raised OF-2430 for this and modified the comment.